src/Settings/Home.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { colors, stylesheet } from '../../config/styles';
+import { StyleSheet } from 'react-native';
+import { Text } from 'react-native-elements';
+import Meteor from 'react-native-meteor';
+import { SafeAreaView } from 'react-navigation';
+import TableView from 'react-native-tableview'
+const { Section, Item } = TableView;
+
+export default class Home extends Component {
+
+	static navigationOptions({ navigation }) {
+		return {
+			headerTitle: 'Settings',
+			headerStyle: {
+				backgroundColor: colors.background
+			},
+			headerTitleStyle: {
+				color: colors.tint
+			},
+			justifyContent: 'center',
+			alignItems: 'center',
+		};
+	}
+	handleSignOut() {
+		Meteor.logout();
+	}
+
+	render() {
+		return (
+			<SafeAreaView style={StyleSheet.absoluteFill}>
+				<TableView
+					style={{ flex: 1 }}
+					tableViewStyle={TableView.Consts.Style.Grouped}
+					tableViewCellStyle={TableView.Consts.CellStyle.Default}>
+					<Section label="General">
+						<Item
+							accessoryType={TableView.Consts.AccessoryType.DisclosureIndicator}
```

Comment:
What is `DisclosureIndicator`?

Reply:
It's this grey chevron thing indicating a push to another view.
https://developer.apple.com/documentation/uikit/uitableviewcellaccessorytype/uitableviewcellaccessorydisclosureindicator

Reply:
<img width="385" alt="screen shot 2018-08-09 at 9 27 46 pm" src="https://user-images.githubusercontent.com/5591523/43934085-78252d20-9c1b-11e8-9fb5-59c77902be9c.png">